test/framework/toolchain.py

```diff
@@ -3144,21 +3145,6 @@ def test_env_vars_external_module(self):
         expected = {}
         self.assertEqual(res, expected)
 
-    def test_get_flag(self):
```

Comment:
Why is this test removed?

Reply:
I think we can keep the test? It's a simpler function of course but can still be tested.

Reply:
makes sense, I consider `get_flag` to be part of the public API